locale/en/messages.json

```diff
@@ -892,15 +892,6 @@
       ]
     ]
   },
-  "Anonymous Bind": {
```

Comment:
Was your plugins-private repo pulled with the latest changes when you extracted? The bind translations should be included (they are from LDAP feature which has been merged)